File: trunk/dotnet/picocontainer/PicoContainer.Test/PicoContainer.Defaults/DefaultLazyInstantiationTestCase.cs
using System;
using NUnit.Framework;
using PicoContainer.Core;
using PicoContainer.Defaults;
using PicoContainer.Tck;

namespace PicoContainer.Defaults
{
	[TestFixture]
	public class DefaultLazyInstantiationTestCase : AbstractLazyInstantiationTestCase
	{
		protected override IMutablePicoContainer createPicoContainer()
		{
			return new DefaultPicoContainer();
		}
	}
}

File: trunk/java/picocontainer/src/test/org/picocontainer/defaults/DefaultLazyInstantiationTestCase.java
package org.picocontainer.defaults;

import org.picocontainer.tck.AbstractLazyInstantiationTestCase;
import org.picocontainer.RegistrationPicoContainer;

/**
 * @author Aslak Helles&oslash;y
 * @version $Revision$
 */
public class DefaultLazyInstantiationTestCase extends AbstractLazyInstantiationTestCase {
    protected RegistrationPicoContainer createRegistrationPicoContainer() {
        return new DefaultPicoContainer.Default();
    }
}
